src/main/java/frc/robot/subsystems/shooter/ShooterFlywheelIOSim.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.subsystems.shooter;
+
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+import frc.robot.Constants;
+
+public class ShooterFlywheelIOSim implements ShooterFlywheelIO
+{
+    private DCMotorSim _upperFlywheelSim  = new DCMotorSim(DCMotor.getNEO(1), 6.75, 0.025);
+    private DCMotorSim _lowerFlywheelSim  = new DCMotorSim(DCMotor.getNEO(1), 6.75, 0.025);
```

Comment:
We're going to need to get constants from the `DCMotor` objects themselves, so let's split those out into their own member variables:

```suggestion
    private DCMotor    _upperMotor        = DCMotor.getNEO(1);
    private DCMotor    _lowerMotor        = DCMotor.getNEO(1);
    private DCMotorSim _upperFlywheelSim  = new DCMotorSim(_upperMotor, 6.75, 0.025);
    private DCMotorSim _lowerFlywheelSim  = new DCMotorSim(_lowerMotor, 6.75, 0.025);
```